web/src/components/CreateMemoRelationDialog.tsx

```diff
@@ -106,7 +116,7 @@ const CreateMemoRelationDialog: React.FC<Props> = (props: Props) => {
           inputValue={searchText}
           value={selectedMemos}
           multiple
-          onInputChange={(_, value) => setSearchText(value.trim())}
+          onInputChange={(_, value) => setSearchText(value)}

```

Comment:
Maybe we can only remove `.trim()` that will fulfill your needs. The other changes don't seem too important.